magento/magento-cloud-patches
# MAGECLOUD-6029: Implement New Relic Logs in Context

### Description

Adds patch to backport ProcessorInterface that was not introduced until Monolog 1.24.0.  ProcessorInterface is necessary to use the New Relic log enricher but Magento 2.1 has a hard requirement on Monolog 1.16.0

### Fixed Issues (if relevant)

1. MCLOUD-6029

### Manual testing scenarios

1. Setup a cloud project with Magento 2.1.16
2. Require magento/magento-cloud-components with the same name
3. Test with current develop branch of ECE-tools
    1. This is necessary as the next release will move the patching to occur before di:compile
Expected result: Deploy occurs successfully

Without this change, the build will fail during di:compile

### Contribution checklist
